mautic/mautic
# Fix link to clicked contact list

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Yes ]
| New feature/enhancement? (use the a.x branch)      | [ no]
| Deprecations?                          | [ no]
| BC breaks? (use the c.x branch)        | [ no]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



Note: this is duplicate of https://github.com/mautic/mautic/pull/11418 that is for M4. This is for M5.

#### Description:
PR for 5.x version. In the Focus Item view, in the "Click count" tab, there is a tracking URL list. In this list In the Unique Click column, there is an invalid link to the contacts that clicked the URL.

Clicking on this link does nothing.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create Focus Item and add link to content
3. Create Landing Page with Focus Item
4. Go to the Landing Page URL and click link in focus item
5. Check Unique Clicks link in the Focus Item view
6. Please note that links are not tracked while you are logged in to the Mautic dashboard.
